Fix Windows Cross compilation failure

Since last 2 days, `make build-go` fails without any changes to zookeeper-operator code, with this error:
```
CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build \
-ldflags "-X github.com/pravega/zookeeper-operator/pkg/version.Version=0.2.3-10 -X github.com/pravega/zookeeper-operator/pkg/version.GitSHA=68bf700" \
-o bin/zookeeper-operator-linux-amd64 cmd/manager/main.go
CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build \
-ldflags "-X github.com/pravega/zookeeper-operator/pkg/version.Version=0.2.3-10 -X github.com/pravega/zookeeper-operator/pkg/version.GitSHA=68bf700" \
-o bin/zookeeper-exporter-linux-amd64 cmd/exporter/main.go
CGO_ENABLED=0 GOOS=darwin GOARCH=amd64 go build \
-ldflags "-X github.com/pravega/zookeeper-operator/pkg/version.Version=0.2.3-10 -X github.com/pravega/zookeeper-operator/pkg/version.GitSHA=68bf700" \
-o bin/zookeeper-operator-darwin-amd64 cmd/manager/main.go
CGO_ENABLED=0 GOOS=darwin GOARCH=amd64 go build \
-ldflags "-X github.com/pravega/zookeeper-operator/pkg/version.Version=0.2.3-10 -X github.com/pravega/zookeeper-operator/pkg/version.GitSHA=68bf700" \
-o bin/zookeeper-exporter-darwin-amd64 cmd/exporter/main.go
CGO_ENABLED=0 GOOS=windows GOARCH=amd64 go build \
-ldflags "-X github.com/pravega/zookeeper-operator/pkg/version.Version=0.2.3-10 -X github.com/pravega/zookeeper-operator/pkg/version.GitSHA=68bf700" \
-o bin/zookeeper-operator-windows-amd64.exe cmd/manager/main.go
# github.com/pravega/zookeeper-operator/vendor/golang.org/x/sys/windows
vendor/golang.org/x/sys/windows/dll_windows.go:21:6: missing function body
vendor/golang.org/x/sys/windows/dll_windows.go:24:6: missing function body
Makefile:31: recipe for target 'build-go' failed
make: *** [build-go] Error 2
The command "make build" exited with 2.
```
This error was seen only with go versions < 1.12.
Hence need to upgrade go version used by zk-operator to 1.12+

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,6 +1,6 @@
 language: go
 go:
-- '1.10'
+- '1.12'
 - tip
 sudo: required
 env:

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,4 +1,4 @@
-FROM golang:1.10.1-alpine3.7 as go-builder
+FROM golang:1.12.10-alpine3.10 as go-builder
 
 ARG PROJECT_NAME=zookeeper-operator
 ARG REPO_PATH=github.com/pravega/$PROJECT_NAME

--- a/README.md
+++ b/README.md
@@ -136,7 +136,7 @@ $ kubectl delete -f deploy/all_ns
 ### Build the operator image
 
 Requirements:
-  - Go 1.10+
+  - Go 1.12+
 
 Use the `make` command to build the Zookeeper operator image.
 
@@ -157,7 +157,7 @@ REPOSITORY                    TAG              IMAGE ID        CREATED         S
 
 pravega/zookeeper-operator    0.1.1-3-dirty    2b2d5bcbedf5    10 minutes ago  41.7MB
 
-pravega/zookeeper-operator    latest           2b2d5bcbedf5    10 minutes ago  41.7MB 
+pravega/zookeeper-operator    latest           2b2d5bcbedf5    10 minutes ago  41.7MB
 
 ```
 Optionally push it to a Docker registry.

--- a/pkg/apis/zookeeper/v1beta1/zookeepercluster_types.go
+++ b/pkg/apis/zookeeper/v1beta1/zookeepercluster_types.go
@@ -13,7 +13,7 @@ package v1beta1
 import (
 	"fmt"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/resource"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
@@ -25,7 +25,7 @@ const (
 
 	// DefaultZkContainerVersion is the default tag used for for the zookeeper
 	// container
-	DefaultZkContainerVersion = "0.2.3"
+	DefaultZkContainerVersion = "latest"
 
 	// DefaultZkContainerPolicy is the default container pull policy used
 	DefaultZkContainerPolicy = "Always"

